cmd/nerdctl/create.go

```diff
@@ -0,0 +1,443 @@
+/*
```

Comment:
Maybe we can put the entire `create.go` into `run.go`. These two commands are not decouplable, and it is hard to find what function belongs to `create.go` and what to `run.go`

Reply:
SGTM, I am also confused about this question : )